src/SFA.DAS.Data.Domain/Models/ApprenticeshipEvent.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using System;
+using SFA.DAS.Events.Api.Types;
```

Comment:
We should get rid of this class, it's utterly pointless and inconsistent.  I would have left it but the automapper config changes made me change my mind! :)